test/unit/components/join-flow-step.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {mountWithIntl} from '../../helpers/intl-helpers.jsx';
+import JoinFlowStep from '../../../src/components/join-flow/join-flow-step';
+
+describe('JoinFlowStep', () => {
+
+    test('test components exist when props present', () => {
```

Comment:
i suggest removing the word "test" here -- it took me a few readings to understand this. As I read the code, the string should just be an assertion of the state when things are working right, so together you'd have `test('only Jedis allowed')`, not `test('test only Jedis allowed')`.

Reply:
done!